activerecord/test/cases/adapters/postgresql/array_test.rb

```diff
@@ -34,14 +34,14 @@ def setup
 
   def test_column
     assert_equal :string, @column.type
-    assert_equal "character varying", @column.sql_type
-    assert @column.array
+    assert_equal "character varying[]", @column.sql_type
```

Comment:
Agree. In the same way there is no reason to remove `[]` there is no reason to keep it. Since we already choose to remove it we should not change.